db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
Hello @sja-thedude , thank you for your review. 
If I could suggest you use port 3001 for the rails server and port 3000 for the react app. I think that’s where the mix up is because everything works perfectly on my machine.